README.md

```diff
@@ -104,10 +104,10 @@ mvn clean verify
 
 If you are a maintainer, you can release a new version by doing the following:
 
-- Merge the changes the need to be released into the `master` branch
+- Merge the changes they need to be released into the `master` branch
```

Comment:
```suggestion
- Merge the changes that need to be released into the `master` branch
```

The old version and your suggested version do not really make sense. :see_no_evil: